Story/Doomwood/Temple.cs

```diff
@@ -24,6 +24,7 @@ public void StoryLine()
         
         Story.PreLoad(this);
 
+        Core.EquipClass(ClassType.Farm);
```

Comment:
put that inside the storyprog

Reply:
I want it to use farm class from the start. Put it on first storyprog is enough?

Reply:
Or on every storyprog? I will do it tomorrow

Reply:
i mean nah if it can contunuue to use the farm its fine 🤷 shouldnt realy have to use these at all

Reply:
Okay, will delete this commit, I don't think it's really needed tho xD

Reply:
Deleted unnecessary commits